core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -69,5 +69,6 @@ class TileImprovement : NamedStats() {
 
     fun hasUnique(unique: String) = uniques.contains(unique)
     fun isGreatImprovement() = hasUnique("Great Improvement")
+    fun cannotBePillaged() = hasUnique("Unpillagable")
```

Comment:
No need to turn this into a function if it only has one use :)
Fold this into the canPillage function